demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -578,10 +579,12 @@ def is_valid_fetch(self) -> bool:
             params = [_key for _key in self.current_file.get('configuration', [])]
             for param in FETCH_REQUIRED_PARAMS:
                 if param not in params:
-                    print_error(f'Integration with fetch-incidents was detected '
-                                f'("isfetch:  true" was found in the YAML file).'
-                                f'\nA required parameter is missing or malformed in the file {self.file_path}, '
-                                f'the param is:\n{param}')
+                    print_error(f'A required parameter'
+                                f'\n```\n{yaml.dump(param)}```\n'
```

Comment:
just the param name, no need the whole dump here
lets remove the backticks
lets remove the line you added (If another other arguments exists in the param remove them.)